src/main/webapp/script.js

```diff
@@ -53,6 +60,117 @@ async function fetchQuestions(page) {
   });
 }
 
+/**
+ * Fetches a single question and its answers from server, 
+ * wraps each in an <li> element, and adds them to the DOM.
+ */
+async function fetchAnswers() {
+  const question_id = (new URL(document.location)).searchParams.get("id");
+  const response = await fetch('/fetch-answers?id=' + question_id);
+  const answersObject = await response.json();
+  const answersContainer = document.getElementById('answers');
+  Object.values(answersObject).forEach(answer => {
+    answersContainer.appendChild(createAnswerElement(answer));
+    const commentsContainer = document.createElement('ul');
+    commentsContainer.setAttribute('class', 'list-group list-group-flush ml-5');
+    answer.commentList.forEach(comment => {
+      if (comment.body) {
```

Comment:
Just curious on how this if statement works. so comment.body is a string, right? So if the string is empty, does it count as false?

Reply:
yes, it is just to avoid adding an empty element